src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/web_socket/LobbyDeltaWebSocketDTO.java

```diff
@@ -0,0 +1,48 @@
+package ch.uzh.ifi.hase.soprafs22.rest.dto.web_socket;
+
+import ch.uzh.ifi.hase.soprafs22.game.player.IPlayer;
+
+import java.util.List;
+
+/**
+ * Delta to be sent through web socket about the lobby so client can process info easier.
+ */
+public class LobbyDeltaWebSocketDTO {
+    private boolean pullUpdate;
```

Comment:
I don't understand what this does. Is this requesting the client to fully pull the lobby?

Reply:
Yes, that some things changed and the client should fetch the lobby again. I was thinking about a good name, but that was the best I could come up with. Do you have any suggestions?